java/ql/src/experimental/CWE-532/SensitiveInfoLog.java

```diff
@@ -0,0 +1,18 @@
+public static void main(String[] args) {
```

Comment:
This has been included by accident I guess?
Ditto the other CWE-532 files.

Reply:
Thanks @intrigus for reviewing this PR. It's weird that I only committed three new files for CWE-939. How did those CWE-532 files show up in the view?

Regards,
@luchua-bc